example/plopfile.js

```diff
@@ -45,19 +45,25 @@ module.exports = function (plop) {
 				name: 'name',
 				message: 'What is your name?',
 				validate: function (value) {
-					if ((/.+/).test(value)) { return true; }
+					if (/.+/.test(value)) {
+						return true;
```

Comment:
return statement should always be in a new line for better readability

Reply:
Please don't make these changes manually. Allow prettier to do whatever prettier wants. I don't want to bikeshed this PR, but I don't want to add further changes that simply follow your code preferences as opposed to what the linter is asking for

Reply:
yes this code changes was done by prettier